front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -128,6 +129,13 @@ export const VaultDataSourceViewContentList = ({
             contentActionsRef={contentActionsRef}
           />
         )}
+        {isWebsite(dataSourceView.dataSource) && (
```

Comment:
Just wondering why this does not take the `contentActionRef` to play with the menu nicely. IIRC, `ContentAction` is the component that handles most of the edition logic.

Reply:
Will do when adding the button on the list view (the ... button). This is for the header of the website detail page only). 👍🏻

Reply:
>This is for the header of the website detail page only). 👍🏻
Then it feels weird that this live in the `VaultDataSourceViewContentList` component 🤔

Reply:
`VaultDataSourceViewContentList ` contains the header (with the search bar & header button), then the data table with content actions. Seems like a legit name

Reply:
Seen IRL.